client/mysqladmin.cc

```diff
@@ -245,7 +245,7 @@ static const char *load_default_groups[]=
 my_bool
 get_one_option(const struct my_option *opt, const char *argument, const char *filename)
 {
-
+  
```

Comment:
please don't add a the whitespace. Not sure of the purpose of adding a new line to the beginning of the function, I can't see this in the coding standard. So please remove line unless a reason for inclusion can be provided.

Reply:
The white space is already there on line 248 in https://github.com/MariaDB/server/blob/11.0/client/mysqladmin.cc. I am removed it and added it back because it was originally there. I am not sure why it is showing I added a new whitespace